doc/pages/user/065-managing-groups.md

```diff
@@ -0,0 +1,65 @@
+# Managing Groups
```

Comment:
This is all good, but please make a preamble that in this documentation astartectl will be used, and API documentation for doing that can be found at <url>